src/app/models/trips.ts

```diff
@@ -20,9 +20,12 @@ export interface ItineraryItem {
   startLocation?: GeoPoint;
   endLocation?: GeoPoint;
   notes?: string;
-  tags?: Tag[];
+  tag: string;
```

Comment:
Does this mean you're just keeping a free-text value in the DB for all items? (i.e. no button to select from a set of tags in the future)

Reply:
Was not quite sure how to handle this so I just dumbed it down for now